README.md

```diff
@@ -76,7 +76,9 @@ The OpenGApps AOSP based build system is placed in `vendor/google/build` and inc
 in `build/core/main.mk`:
 ```
 # Include the google-specific config
+ifneq ($(GAPPS_VARIANT),)
```

Comment:
This is a reference to existing code in AOSP...not something that should be modified.